requirements.txt

```diff
@@ -13,7 +13,7 @@ pytest==7.4.3
 setuptools~=66.1.1
 scipy~=1.10.1
 torch~=1.13.1
-tensorboard
+tensorboard==2.13.0
```

Comment:
Is this one of the more recent versions?

Reply:
no, current version is `2.18.0`. version `2.13.0` was the last one with python 3.8 support